src/tests/simple.test.js

```diff
@@ -0,0 +1,21 @@
+import { assert } from 'chai';
+
+import add from '../controllers/add';
+import first from '../controllers/first';
```

Comment:
What happened to the test client?
Please create and export a general setting for tests that will be imported in all subsequent test files